endpoints/api/policy.py

```diff
@@ -0,0 +1,323 @@
+import logging
+
+from flask import request
+
+import features
+from auth.auth_context import get_authenticated_user
+from auth.permissions import AdministerOrganizationPermission
+from data import model
+from endpoints.api import (
+    ApiResource,
+    allow_if_superuser,
+    path_param,
+    request_error,
+    require_user_admin,
+    resource,
+    show_if,
+    validate_json_request,
+)
+from endpoints.exception import InvalidRequest, NotFound, Unauthorized
+
+logger = logging.getLogger(__name__)
+
+
+@resource("/v1/organization/<orgname>/autoprunepolicy/")
+@path_param("orgname", "The name of the organization")
+@show_if(features.AUTO_PRUNE)
+class OrgAutoPrunePolicies(ApiResource):
+    """
+    Resource for listing and creating organization auto-prune policies
+    """
+
+    schemas = {
+        "AutoPrunePolicyConfig": {
+            "type": "object",
+            "description": "The policy configuration that is to be applied to the organization",
+            "required": ["method", "value"],
+            "properties": {
+                "method": {
+                    "type": "string",
+                    "description": "The method to use for pruning tags (number_of_tags, creation_date)",
+                },
+                "value": {
+                    "type": ["integer", "string"],
+                    "description": "The value to use for the pruning method (number of tags e.g. 10, time delta e.g. 7d (7 days))",
+                },
+            },
+        },
+    }
+
+    def get(self, orgname):
+        permission = AdministerOrganizationPermission(orgname)
+        if not permission.can() and not allow_if_superuser():
+            raise Unauthorized()
+
+        policies = model.autoprune.get_namespace_autoprune_policies_by_orgname(orgname)
+
+        return {"policies": [policy.get_view() for policy in policies]}
+
+    @validate_json_request("AutoPrunePolicyConfig")
```

Comment:
`@require_scope(scopes.ORG_ADMIN)` decorator should be used here, as well as the other endpoints

Reply:
added to the org endpoints. The user endpoints are already using `@require_user_admin()`